infra/ansible/publish_torchprime_e2e_test_docker.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This script builds and pushes a docker image to be used for torchprime E2E tests.
```

Comment:
Document

- where the image is pushed to,
- what the retention policy is, and
- how to change the retention policy

?